src/viur/core/skeleton.py

```diff
@@ -1927,7 +1926,13 @@ def processRemovedRelations(removedKey: db.Key, cursor=None):
 
 
 @CallDeferred
-def updateRelations(destKey: db.Key, minChangeTime: int, changedBone: t.Optional[str], cursor: t.Optional[str] = None):
+def updateRelations(
+    dest_key: db.Key,
+    min_change_time: int,
+    changed_bones: t.Optional[list[str] | str] = None,
```

Comment:
```suggestion
    changed_bones: t.Optional[t.Iterable[str] | str] = (),
```
Default to empty tuple as well...